kernel/poll.c

```diff
@@ -142,38 +150,72 @@ static long _syscall_poll(struct pollfd* fds, nfds_t nfds, int timeout)
         }
     }
 
-    /* pre-poll for kernel events */
+    myst_syscall_clock_gettime(CLOCK_MONOTONIC, &start);
+    if ((original_timeout > 500) || (original_timeout < 0))
+        timeout = 500;
```

Comment:
Where is 500 coming from? Is this a Linux convention?

Reply:
it is a time that I thought would be acceptable!